### DISCUSSION TITLE
Welcome to Multi-Cloud-Transfer-Tool Discussions!

### DISCUSSION THREAD

**Discussion by TheCaduceus** [TheCaduceus/Multi-Cloud-Transfer-Tool#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by SenpaiNZO** [comment]
Hey Dr.Caduceus. I'm very glad to see that you're still available here cause I needed to show you an error I'm facing with you telegram uploader code. 

The thing is that when I send some files the filename instead of appearing like "@Geeksenpai.Power.Book.III.Raising.Kanan.S04E05.FRENCH.WEBRip.x264.mp4" appear like "@Geeksenpai_Power_Book_III_Raising_Kanan_S04E05_FRENCH_WEBRip_x264.mp4". It really bother me and I thought you can help me resolve the problem am facing here

And thank for the tool, it's very useful for us. ✨❤️
